lib/logger/lib/logger.ex

```diff
@@ -19,6 +19,27 @@ defmodule Logger do
     * Wraps OTP's `error_logger` to prevent it from
       overflowing.
 
+  A common use case is tracking when an important function is called.
```

Comment:
I don't think we should say this. I don't think "tracking when an important function is called" is a particularly common use case of `Logger` (or at least, I think it's the more general "tracking when an event of interest happens in your system").